src/main/java/com/netchill/db/dao/movie/GenreDao.java

```diff
@@ -26,7 +26,7 @@ public Optional<Genre> getGenreById(Long id) {
                 .fetchOne());
     }
 
-    public List<Genre> getPreviewGenres() {
+    public List<Genre> featuredGenre() {
```

Comment:
```suggestion
    public List<Genre> fetchFeaturedGenre() {
```

Reply:
This is a DAO method, so its name must be prefixed with fetch.
```suggestion
    public List<Genre> fetchFeaturedGenre() {
```